server/src/main/java/org/opensearch/rest/action/admin/cluster/RestDecommissionAction.java

```diff
@@ -49,6 +49,9 @@ DecommissionRequest createRequest(RestRequest request) throws IOException {
         DecommissionRequest decommissionRequest = Requests.decommissionRequest();
         String attributeName = request.param("awareness_attribute_name");
         String attributeValue = request.param("awareness_attribute_value");
-        return decommissionRequest.setDecommissionAttribute(new DecommissionAttribute(attributeName, attributeValue));
+        decommissionRequest.setDecommissionAttribute(new DecommissionAttribute(attributeName, attributeValue));
+        return decommissionRequest.setDrainingTimeout(
+            request.paramAsTime("draining_timeout", DecommissionRequest.DEFAULT_NODE_DRAINING_TIMEOUT)
```

Comment:
Since the system isn't doing an explicit draining, should we call this as `delay_timeout` and also set a query param that says ?no_delay=true which sets the timeout at 0s

Reply:
Done.